lib/handlers/quote/schema/quote-schema.ts

```diff
@@ -57,6 +57,7 @@ export const QuoteQueryParamsJoi = Joi.object({
   permitSigDeadline: Joi.number().optional(),
   // TODO: Remove once universal router is no longer behind a feature flag.
   enableUniversalRouter: Joi.boolean().optional().default(false),
+  quoteSpeed: Joi.string().valid('fast', 'standard', 'patient').optional().default('standard'),
```

Comment:
patient is a bit of an unusual choice :D 

slow feels more standard..

Reply:
yeah, I actually think I wont encode any slower speed than 'standard', so for now only 'fast' and 'standard'